src/apps/testapps/testVec2d.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Uber Technologies, Inc.
```

Comment:
This should've been 2018.

Reply:
True. Never known how to manage this, but presumably if we care we ought to run an annual script to update.

Reply:
OK so I thought the same but @yurishkuro advised me that the ideal solution is to update files you touch and instead of just replacing use this pattern `Copyright 2015, 2018` (if a range of years, I just do `2017-2018`). See this pull request in Jaeger to avoid blindly replacing year in all files https://github.com/jaegertracing/jaeger/commit/d7d1c74fa74f77d2c7faa50a0c568f979d9a30fd.

Reply:
In this case the entire file was being added, so just 2018 would be correct.

Personally I find copyright notices in source code anachronistic in the age of Git. You have the full history of what was edited by whom and the LICENSE file to tell you what rules apply.

Reply:
OK I see. And yes completely agree. IDK how legal feels about that but personally makes no sense. I think the fear is for those that copy a source file directly out of a project and use it in their own work. Seems like an unjustified fear.